src/EventStore.ClientAPI/ClusterDnsSeed.cs

```diff
@@ -0,0 +1,36 @@
+using System.Net;
+using EventStore.ClientAPI.Common.Utils;
+using EventStore.ClientAPI.Transport.Http;
+
+namespace EventStore.ClientAPI {
+	public class ClusterDnsSeed: IGossipSeed {
+		private readonly string _clusterDns;
+		private readonly bool _seedOverTls;
+		private readonly bool _v20HostHeader;
+		private readonly int _port;
+
+		public ClusterDnsSeed(string clusterDns, int port, bool seedOverTls = false, bool v20HostHeader = false) {
+			_clusterDns = clusterDns;
+			_seedOverTls = seedOverTls;
+			_v20HostHeader = v20HostHeader;
+			_port = port;
+		}
+
+		public string ToHttpUrl() {
+			if (_v20HostHeader) {
+				var scheme = _seedOverTls ? "https" : "http";
```

Comment:
Use `EndpointExtensions.HTTPS_SCHEMA` & `EndpointExtensions.HTTP_SCHEMA` as further down in the method?

Reply:
Agree